tests/test_imageDecorrelation.py

```diff
@@ -449,7 +449,7 @@ def testNoiseDiffimCorrection(self):
 
         self._setUpSourcelessImages(svar=svar, tvar=tvar)
         diffExp, mKernel, expected_var = self._makeAndTestUncorrectedDiffim()
-        corrected_diffExp = self._runDecorrelationTask(diffExp, mKernel)
+        corrected_diffExp = self._runDecorrelationTask(diffExp.clone(), mKernel)
```

Comment:
Just for my edification, why the change to .clone() here?

Reply:
Other places too, actually....

Reply:
That's so I can modify the difference exposure in place in the decorrelation task, and not effect the tests.